src/esp/assets/ResourceManager.cpp

```diff
@@ -475,8 +484,20 @@ bool ResourceManager::createScene(Importer& importer,
     for (auto objectID : sceneData->children3D()) {
       createObject(importer, info, metaData, sceneNode, drawables, objectID);
     }
+  } else if (importer.mesh3DCount() && meshes_[metaData.meshIndex.first]) {
+    // no default scene --- standalone OBJ/PLY files, for example
+
+    // create scene parent node with transformation aligning to global frame
+    auto& sceneNode = parent.createChild();
+    const quatf transform =
+        quatf::FromTwoVectors(info.frame.front(), geo::ESP_FRONT);
```

Comment:
This should probably be `quatf::FromTwoVectors(info.frame.up(), geo::ESP_UP);` instead. We explicitly define the up vector for most of our assets, but leave front to the default in many cases.

Reply:
This is verbatim copied from the branch above (see line 480), should I change both?

Reply:
Ah! Let's leave the current behavior in place for both branches but add a "TODO ought to align with ESP_UP" above them.

Reply:
#61 by @erikwijmans fixes that, so I propose merging that one first -- and then I update the above to be the same as done in #61.